packages/backend/code/server.ts

```diff
@@ -2,13 +2,17 @@ import express from "express";
 import cors from "cors";
 import http from "http";
 import { Server } from "socket.io";
+import { users } from "./index";
+import { hashSync, compareSync } from "bcryptjs";
+import { User } from "./types";
 
+let jwt = require("jsonwebtoken");
```

Comment:
I think this can be an ES6 import

```suggestion
import jwt from "jsonwebtoken";
```

Reply:
You may need to install the typings as well, it should say how if you hover the error after making the change.